validation/ref/numeric/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# note: function add_validation_data defined in validation/CMakeLists.txt
+
+add_validation_data(
+    OUTPUT numeric_soma.json
+    DEPENDS numeric_soma.jl HHChannels.jl
+    COMMAND julia numeric_soma.jl)
```

Comment:
Ah, we (er, I) don't, no. Good point. We also depend on some Julia packages — I'll see how hard it would be to have CMake complain usefully if they are not installed.